lib/measure_repository_service_test_kit/measure_data_requirements.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require 'json'
+require_relative '../utils/assertion_utils'
+
+module MeasureRepositoryServiceTestKit
+  # tests for $data-requirements operation for Measure service
+  # rubocop:disable Metrics/ClassLength
+  class MeasureDataRequirements < Inferno::TestGroup
+    # module for shared code for $data-requirements assertions and requests
+    module DataRequirementsHelpers
```

Comment:
Unsure if this was discussed already, but might be a good idea for future proofing re: @lmd59's task if we make this helper available outside of just the measure data requirements group since the same assertion will likely be re-used almost exactly in that other test group as well